plugins/hydra_ray_launcher/tools/setup_ami.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Are we expecting users to need to run this?

Reply:
Thx for taking a look Omry.  Yes, the tests have been consistently passing, I think they are ready to be landed, we can talk more about this in person when we meet. 

As for the `setup_ami.py` tools, they are solely needed for integration tests to get around the restrictions we put on the instances (no outbound traffic at all for the test EC2 instances so we need to pre-install all the stuff.) I will add more details to the tools documentation/naming to make that clear. The users should be able to use whatever AMI they want and they are not expected to need to build AMI just so they can use the RAY launcher. 

If it's OK with you I want to follow up separately on the example app. It's easier to focus on the app in a smaller PR.

Another follow up I'm planning on doing is to automate the building of the AMI  for integration tests (make it a nightly job.)

Reply:
Sure, we can postpone the example app updates for later.
Recursive defaults is getting pretty close to landing, so you will be able to take advantage of it soon.

Please respond to the rest of my comments.

Reply:
Yes I'm in the process of updating the code to address you and Shagun's comments :)